### PR TITLE
Update Quick start to use containerized sdk

### PR DESCRIPTION
**Pull request by scanon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,56 +48,36 @@ Once your account is approved, contact us with your username and ask to be added
...
-Get the SDK:
-
-    git clone https://github.com/kbase/kb_sdk
 
-Pull dependencies and configure the SDK:
+Pull the KBase SDK image.
 
-    cd kb_sdk
-    make bin
+    docker pull kbase/kb-sdk
```
Comment (kkellerlbl) [inline README.md]: Is this the right image name/tag?  Maybe @sychan knows

**Comment by kkellerlbl** [comment]
Are we ready to merge to master?

File: README.md [hunk 2]
```diff
@@ -48,56 +48,36 @@ Once your account is approved, contact us with your username and ask to be added
...
-    export PATH=$(pwd)/bin:$PATH
-    source src/sh/sdk-completion.sh
+    kb-sdk sdkbase
 
 Test installation:
 
     kb-sdk help
 
 
-#### Build from source
```
Comment (MrCreosote) [inline README.md]: This should probably live somewhere, maybe in an apocryphal appendix

Comment (scanon) [reply README.md]: I could make a separate PR for that.

File: README.md (hunk 1 above)
Comment (scanon) [reply README.md]: Just to clarify.  This isn't the SDK Base image that Steve is working on.  But we probably do want to sync these things up.  We could push up a new tag for this image (e.g. kbase/kb-sdk:2) that would use the new base images.

**Comment by sychan** [comment]
Looks fine to me.

File: README.md [hunk 3]
```diff
@@ -48,56 +48,36 @@ Once your account is approved, contact us with your username and ask to be added
...
-Pull dependencies and configure the SDK:
+Pull the KBase SDK image.
 
-    cd kb_sdk
-    make bin
+    docker pull kbase/kb-sdk
 
-Download the local KBase SDK base Docker image:
+Create a wrapper script to call the SDK.  This script can be placed anywhere you wish, but it should be
+on your path.  In this example, we will install it in $HOME/bin/.  To use this approach on Windows, you may need to install the Windows for Linux (WSL).
```
Comment (MrCreosote) [inline README.md]: extra `the`

File: README.md [hunk 4]
```diff
@@ -48,56 +48,36 @@ Once your account is approved, contact us with your username and ask to be added
...
+    chmod 755 $HOME/bin/kb-sdk
+    export PATH=$PATH:$HOME/bin/kb-sdk
 
-Add the kb-sdk tool to your PATH and enable command completion.  From the kb_sdk directory:
+Download the local KBase SDK base Docker image:
 
-    # for bash
-    export PATH=$(pwd)/bin:$PATH
-    source src/sh/sdk-completion.sh
+    kb-sdk sdkbase
```
Comment (MrCreosote) [inline README.md]: Should this be sdkbase2? Not sure what sdkbase means here

File: README.md (hunk 2 above)
Comment (MrCreosote) [reply README.md]: Still think this should be moved to an appendix in these docs

File: doc/building_sdk.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+#### Building SDK
```
Comment (MrCreosote) [inline doc/building_sdk.md]: is this linked from anywhere? Should it be?

Comment (MrCreosote) [inline doc/building_sdk.md]: maybe add the build from source instructions here

Comment (scanon) [reply doc/building_sdk.md]: Yes.  I had done that and not pushed.

Comment (scanon) [reply doc/building_sdk.md]: Yes. I had done this but not pushed it.

Comment (scanon) [reply doc/building_sdk.md]: Not yet.

File: README.md (hunk 4 above)
Comment (scanon) [reply README.md]: It just triggers the pull.  We could use something like pullbase.

File: doc/building_sdk.md (hunk 5 above)
Comment (scanon) [reply doc/building_sdk.md]: addressed

File: README.md (hunk 4 above)
Comment (MrCreosote) [reply README.md]: yeah, I think a name that better describes what the command does (`pull-base-image` is exact but verbose) would be helpful.

File: doc/building_sdk.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    # for bash
+    export PATH=$(pwd)/bin:$PATH
+    source src/sh/sdk-completion.sh
+
+
+#### Build from source
+
+Additional System Dependencies:
+
+- Java JDK 7 or 8 (9 is currently incompatible) http://www.oracle.com/technetwork/java/javase/downloads/index.html
```
Comment (MrCreosote) [inline doc/building_sdk.md]: If this is in addition to the list above, java is already in that list

File: README.md (hunk 2 above)
Comment (scanon) [reply README.md]: Done

**Pull request merged**